tests/test_dia_object.py

```diff
@@ -20,28 +20,141 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
+import numpy as np
 import unittest
+
+from lsst.ap.association import *
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
 import lsst.utils.tests
 
 
-class TestDIAObject(unittest.TestCase):
+def create_test_dia_sources(n_sources=5):
+    """ Create dummy DIASources for use in our tests.
 
-    def setUp(self):
-        pass
+    Parameters
+    ----------
+    n_sources : int (optional)
+        Number of fake sources to create for testing.
 
-    def tearDown(self):
-        pass
+    Returns
+    -------
+    A lsst.afw.SourceCatalog
+    """
+    sources = afwTable.SourceCatalog(afwTable.SourceTable.makeMinimalSchema())
 
-    def test_initialization(self):
-        pass
+    for src_idx in range(n_sources):
+        src = sources.addNew()
+        src['id'] = src_idx
+        src['coord_ra'] = afwGeom.Angle(0.0 + 1. * src_idx,
+                                        units=afwGeom.degrees)
+        src['coord_dec'] = afwGeom.Angle(0.0 + 1. * src_idx,
+                                         units=afwGeom.degrees)
+        # Add a flux at some point
 
-    def test_dia_source_append(self):
-        pass
+    return sources
 
-    def compute_light_curve(self):
-        pass
 
-    def compute_summary_statistics(self):
+class TestDIAObject(unittest.TestCase):
+
+    def setUp(self):
+        """ Create the DIAObjct schema we will use throughout these tests.
+        """
+        self.dia_obj_schema = make_minimal_dia_object_schema()
```

Comment:
Correct. This was left over from when before I had the create_test_dia_sources method in the test. I have removed the setUp and tearDown methds.